keycloak/keycloak-admin-ui
# Identity providers(mappers): update form fields for all Social mapper types

## Motivation

Closes #1206 

## Verification Steps


1. Go to Identity Providers
2. Create a provider and select a "Social" identity provider from the . 
3. Go to the "Mappers" tab
4. Click "Create a new mapper."
5. Verify that when a new mapper type is selected in the "Mapper Type" menu, the fields in the form change corresponding to the mapper type. 
6. Create a mapper for every type and verify the save is successful.

## Checklist:


## Additional Notes
